tests/components/mqtt/test_water_heater.py

```diff
@@ -744,24 +749,7 @@ async def test_temperature_unit(
                     },
                 ),
             ),
-            UnitOfTemperature.KELVIN,
```

Comment:
Same here: KELVIN is not a valid value for unit system temperature unit, so I don't think it makes sense to try to keep this test

Reply:
Agree